master/internal/rm/kubernetesrm/pods.go

```diff
@@ -194,10 +200,16 @@ func Initialize(
 	check.Panic(check.True(ok, "pods address already taken"))
 	s.Ask(podsActor, actor.Ping{}).Get()
 
+	p.startPodInformer(s)
+	p.startNodeInformer()
```

Comment:
little hard to reason about if starting these here is correct. i noticed `startPodInformer` appends `p.informers = append(p.informers, i)` but this is happening outside of the lock while the actor has already been started. but i also didn't see p.informers or p.nodeInformer used anywhere. I would just move the `p.start...` calls back to `actor.PreStart` until `pods` is refactored, unless it needs to be here.

Reply:
`informers` and `nodeInformers` were used in `actor.ChildFailed` case, but I'm removing the structs as we have error handling within the pod/node informer modules now